pages/dkp/kaptain/1.2.0-1.1.0/troubleshooting/kaptain-1.1.0/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: layout.pug
+navigationTitle: Troubleshooting Kaptain 1.0
+title: Troubleshooting Kaptain 1.0
+menuWeight: 10
+excerpt: View troubleshooting information for Kaptain 1.1.0
```

Comment:
This page is already 'versioned' higher up, do we need to include the version here as well?  It is confusing (and makes more work for us when releasing a new version) to have the version number in the title of the page here.

```suggestion
navigationTitle: Troubleshooting Kaptain
title: Troubleshooting Kaptain 
menuWeight: 10
excerpt: View troubleshooting information for Kaptain
```